src/lib/components/global/TextInput.svelte

```diff
@@ -15,6 +15,7 @@
     {placeholder}
     {disabled}
     on:input
+    on:keydown|stopPropagation
```

Comment:
Do we want to do this for all `TextInput`s or should I add a flag prop to toggle `stopPropagation` on events on specific instances of `TextInput`?

Reply:
If this resolves the piano playing while typing issue then yes do it for all components

Reply:
Yeah the `stopPropagation` event method stops the event from bubbling past the call, so in this case this prevents `keydown` events from bubbling up to the window.  
https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation